### PR TITLE
CDS-209 Add company Area to investments projects search CSV 

### PR DESCRIPTION
**Pull request by Staberinde**
### Description of change

* Add company Area to investments projects search CSV via investor_company__address_area behind feature flag
* Add feature-flag for area on company search CSV export

### Checklist

* [x] If this is a releasable change, has a news fragment been added?

  <details>
  <summary>Explanation</summary>
  
  A news fragment is required for any releasable change (i.e. code that runs in or affects production) so that a corresponding changelog entry is added when releasing.
  
  Check [changelog/README.md](https://github.com/uktrade/data-hub-api/blob/master/changelog/README.md) for instructions.
  
  </details>
  
* [x] Has this branch been rebased on top of the current `develop` branch?

  <details>
  <summary>Explanation</summary>
  
  The branch should not be stale or have conflicts at the time reviews are requested.
  
  </details>

* [x] Is the CircleCI build passing?

### General points

<details>
<summary><strong>Other things to check</strong></summary><p></p>

* Make sure `fixtures/test_data.yaml` is maintained when updating models
* Consider the admin site when making changes to models
* Use select-/prefetch-related field lists in views and search apps, and update them when fields are added
* Make sure the README is updated e.g. when adding new environment variables

</details>

See [docs/CONTRIBUTING.md](https://github.com/uktrade/data-hub-api/blob/develop/docs/CONTRIBUTING.md) for more guidelines.

### CODE REVIEW

File: datahub/search/company/views.py [hunk 1]
```diff
@@ -206,18 +207,36 @@ class SearchCompanyExportAPIView(SearchCompanyAPIViewMixin, SearchExportAPIView)
...
+            'name': 'Name',
+            'link': 'Link',
+            'sector_name': 'Sector',
+            'address_country__name': 'Country',
+        }
+        field_titles.update(
+            {
+                'address_area__name': 'Area',
+            }
+            if is_feature_flag_active('address-area-company-search')
```
Comment (ambrozic) [inline datahub/search/company/views.py:229]: how about:
```
if is_feature_flag_active('address-area-company-search'):
    field_titles.update({...})
```
this way you do not update `field_titles` if condition is not true

File: datahub/search/company/views.py [hunk 2]
```diff
@@ -206,18 +207,36 @@ class SearchCompanyExportAPIView(SearchCompanyAPIViewMixin, SearchExportAPIView)
...
+            else {},
+        )
```
Comment (tomkingston) [inline datahub/search/company/views.py:231]: Wouldn't this be simpler (and shorter) as:
```
if is_feature_flag_active('address-area-company-search':
    field_titles['address_area__name'] = 'Area'
```

**Review by tomkingston — APPROVED** [review]
Just noticed I appear to have pretty much the same comment as Gregor!

**Review by ambrozic — APPROVED** [review]

File: datahub/search/company/views.py (hunk 2 above)
Comment (Staberinde) [reply datahub/search/company/views.py]: Yeah that would be another way of achieving the same functionality. The reason why I went with `update` is that it's used for the following statement too, and using it here keeps the functionality to consistent. Happy to change it if you feel strongly though

Comment (Staberinde) [reply datahub/search/company/views.py]: Also I've made the change to using an outer `if` statement off the back of Gregor's suggestion anyway

Comment (tomkingston) [reply datahub/search/company/views.py]: Yeah, update is fine - main thing was just the same as Gregor's comment with the if

**Pull request merged**